backend/src/main/scala/cromwell/backend/BackendInitializationData.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.backend
```

Comment:
Thanks. I was just saying to myself the other day, "I wish this was in its own file"